setup.py

```diff
@@ -2,34 +2,40 @@
 
 from setuptools import setup, find_packages
 
+DESCRIPTION = 'A python toolbox for analyzing and plotting free recall data'
+LONG_DESCRIPTION = """\
+Quail is a Python package that facilitates analyses of behavioral data from memory experiments. (The current focus is on free recall experiments.) Key features include:
+
+- Serial position curves (probability of recalling items presented at each presentation position)
+Probability of Nth recall curves (probability of recalling items at each presentation position as the Nth recall in the recall sequence)
+- Lag-Conditional Response Probability curves (probability of transitioning between items in the recall sequence, as a function of their relative presentation positions)
+- Clustering metrics (e.g. single-number summaries of how often participants transition from recalling a word to another related word, where "related" can be user-defined.)
+- Many nice plotting functions
+- Convenience functions for loading in data
+- Automatically parse speech data (audio files) using wrappers for the Google Cloud Speech to Text API
+
+The intended user of this toolbox is a memory researcher who seeks an easy way to analyze and visualize data from free recall psychology experiments.
+"""
+
+with open('requirements.txt') as f:
+    REQUIREMENTS = f.read().splitlines()
+
+EXTRAS_REQUIRE={
```

Comment:
@jeremymanning most of the files that changed were documentation-related files. the only new functional thing is the way i've set up the optional dependencies here:

Reply:
by default, EL specific and speech decoding libraries are not installed. this will make maintaining a stable package easier down the road. if you want those additional features, it's easy to install them: `pip install quail[speech-decoding, efficient-learning]`

Reply:
that's a good idea.  re: the efficient-learning dependencies, is it documented somewhere (e.g. in the EL data repo) that in order to analyze those data with quail we need this new install instruction?

Reply:
ah, no but i'll add it to the readme!